msprime/simulations.py

```diff
@@ -599,6 +606,14 @@ class RecombinationMap(object):
     position must be zero, and by convention the last rate value
     is also required to be zero (although it does not used).
 
```

Comment:
addressing #549 here

Reply:
Hmm, looks like the "Closes" bit needs to either be in the commit message or in the PR description.